### DISCUSSION TITLE
Intune Remediation Script Error

### DISCUSSION THREAD

**Discussion by ghost** [Set-OutlookSignatures/Set-OutlookSignatures#106] [Q&A] [answered]
I am having issues with the Intune remediation script. I continue to get the following error.

"Problem connecting to or reading from folder 'C:\Users\Public\Documents\signature\xxx'. Exit."

Any ideas on this would be appreciated.

Thank you.

**Comment by GruberMarkus** [comment]
Hi @Paul-KL,

Unfortunately, the error message is ambiguous due to missing context.

Please provide the full script output in text format for further analysis.

**Comment by ghost** [comment]
Thanks for the quick response. Attached is the log file. 
[logfile_v1.txt](https://github.com/Set-OutlookSignatures/Set-OutlookSignatures/files/14779517/logfile_v1.txt)

**Comment by GruberMarkus** [comment, marked as answer]
You are passing the BenefactorCircleLicenseFile parameter with the value 'xxx', but you are not a Benefactor Circle client and therefore do not have such a file.

Do not pass the parameters BenefactorCircleLicenseFile and BenefactorCircleLicenseId.

Please let me know if this solves your problem.

**Reply by ghost** [reply]
That was it! That was a silly oversight on my part. Thank you so much for the quick response and the awesome solution.

**Reply by GruberMarkus** [reply]
My pleasure!
